tools/codegen/cmd/featuregate-test-analyzer.go

```diff
@@ -589,15 +601,12 @@ func listTestResultForVariant(featureGate string, jobVariant JobVariant) (*Testi
 			// example: release-4.18, release-4.17
 			release = strings.TrimPrefix(currentRelease, "release-")
 		} else {
-			// Temporary until branching on 2025-04-18, or we find a way to fix sippy to return the dev branch sooner.
-			release = "4.19"
-
 			// means its main branch
-			// var err error
-			// release, err = getLatestRelease()
-			// if err != nil {
-			// 	return nil, fmt.Errorf("couldn't fetch latest release version: %w", err)
-			// }
+			var err error
+			release, err = getLatestRelease()
+			if err != nil {
+				return nil, fmt.Errorf("couldn't fetch latest release version: %w", err)
+			}
```

Comment:
Could we get an output somewhere, that tells us what release it's using as the base to fetch results? Maybe a log line, or somewhere in the output markdown, I don't if that would be too difficult?

Reply:
Added release logging, updated references from `component readiness` to `sippy` since these aren't actually CR endpoints.  Wasn't looking to turn this into a refactor but it doesn't look like the release logic needs to be in the loop, there is only one env setting or one latest release...